cloudbuild.yaml

```diff
@@ -1,46 +0,0 @@
-steps:
-  - id: 'clone dev services'
-    name: gcr.io/cloud-builders/git
-    entrypoint: bash
-    secretEnv:
-      - GITHUB_TOKEN
-    args:
-      - '-c'
-      - |
-        git clone https://$$GITHUB_TOKEN@github.com/Yesware/dev-services.git
-        cd dev-services
-        echo -e "networks:\n  default:\n    external:\n      name: cloudbuild" >> docker-compose.yml
-  - id: 'start dev services'
-    name: docker/compose
-    args: ['-f', 'dev-services/docker-compose.yml', 'up', '-d', '--build']
-  - id: 'bundle install'
-    name: gcr.io/repcore-prod/yw-ci:latest
-    env:
-      - 'RAILS_ENV=test'
-    script: |
-      #!/usr/bin/env bash
-      source /nvm/nvmrc
-      bundle install
-    waitFor: ['-']
-    volumes:
-      - name: 'bundle'
-        path: '/usr/local/bundle'
-  - id: 'rake ci'
-    name: gcr.io/repcore-prod/yw-ci:latest
-    env:
-      - 'RAILS_ENV=test'
-      - 'CLOUDBUILD_PG=postgres-14-7'
-      - 'CLOUDBUILD_REDIS_URL=redis://redis-6-2:6379'
```

Comment:
We should also remove the references to `CLOUDBUILD_REDIS_URL` in [spec/settings_spec.rb](https://github.com/vendasta/settings-in-redis/pull/6/files#diff-7e5c6ab15383888d52226c93f01112992f27a669de8f9a179b42227495a11c3e)